active/ProgressBarWrapping/ProgressBarWrapping.md

```diff
@@ -0,0 +1,77 @@
+
+# Background
+
+The Xaml [ProgressBar](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.ProgressBar) control is used to show progress of an operation, where the typical visual appearance is a bar that animates a filled area as progress continues. (There's also a [ProgressRing](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ProgressRing) control that shows a circle rather than a line.)
+
+The update in this spec is for a feature to make the bar appear such that the rectangle wraps around from one side to the other (when in indetermine mode). This is implemented by having two rectangles, so during the "wrap" what you see is actually a partial rectangle on the right and another partial one on the left:
+
+![image](./IndeterminatePB.gif)
+
+The current single rectangle is controlled by two read-only properties, :  
+`ContainerAnimationStartPosition` and  
+`ContainerAnimationEndPosition`. 
+
+These are exposed by the ProgressBar's [TemplateSettings](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ProgressBar.TemplateSettings), and consumed by the control template. 
+
+The new APIs are a second pair of these properties to similarly control the second rectangle:  
+`ContainerAnimationStartPosition2` and  
+`ContainerAnimationEndPosition2`
+
+Naming note: the "2" suffix looks like a version indicator, which would be an unusual naming pattern. But it's actually referring to the second rectangle. So it would be better to rename the existing properties with a "1" suffix, similar to [ScaleAnimation.Control1/2](https://docs.microsoft.com/uwp/api/Windows.UI.Core.AnimationMetrics.ScaleAnimation.Control1), but that would be a breaking change to existing API.
+
+
+# Examples
+
+From the ProgressBar's new built-in control template, the following is used to animate the second rectangle by binding to the `ContainerAnimationStartPosition2` and `ContainerAnimationEndPosition2` properties:
+
+```Xml
+<DoubleAnimationUsingKeyFrames
+    Storyboard.TargetName="IndeterminateProgressBarIndicator2"
+    Storyboard.TargetProperty="(UIElement.RenderTransform).(CompositeTransform.TranslateX)">
+
+    <SplineDoubleKeyFrame KeyTime="0" 
```

Comment:
Good point! After some investigation, do you think it makes sense to make it a DiscreteDoubleKeyFrame? Since they are temporary holder positions, and no actual x-translation is done.

Reply:
Agreed, it only needs to be a discrete key frame.

Reply:
https://github.com/microsoft/microsoft-ui-xaml/issues/2738